drivers/intel/soc/stratix10/io/s10_memmap.c

```diff
@@ -0,0 +1,251 @@
+/*
```

Comment:
This should be in plat/intel as this is not a driver, but platform enablement right?
Also let's rename this to something qspi related, such as s10_mmap_qspi.c as this is a qspi memory mapped io driver.

Reply:
This is for enablement specific to intel platform which based on io driver. By referring to renesas, I leave the code inside the source directory. Will move it if plat/intel is more appropriate for this. Also, ill rename the driver to be more descriptive.

Reply:
@antonio-nino-diaz-arm, what do you think? Should this be in plat/ or driver/?

Reply:
We do have a situation that there are platform enablement drivers currently in `drivers/` folder. This was a recent policy change to make the drivers more visible to encourage reuse between platforms (with proper abstractions introduced) rather than having them hidden deep inside `plat/` folder.

Hence this can be left in the current location in my opinion.

Reply:
Thanks for the input

Reply:
Noted with thanks @soby-mathew